dlc-manager/src/sub_channel_manager.rs

```diff
@@ -2840,282 +2840,255 @@ where
         msgs
     }
 
-    /// Updtates the view of the blockchain processing transactions and acting upon them if
+    /// Updates the view of the blockchain processing transactions and acting upon them if
```

Comment:
Maybe might as well update the comment, IIUC this is not updating the view of the blockchain anymore since it's done separately?

Reply:
Good catch, I clearly didn't read what I updated lol.